src/commands/node.mjs

```diff
@@ -502,53 +506,125 @@ export class NodeCommand extends BaseCommand {
       {
         title: 'Enable mirror node',
         task: async (ctx, parentTask) => {
-          const subTasks = [
-            {
-              title: 'Get the mirror node importer address book',
-              task: async (ctx, _) => {
-                ctx.addressBook = await self.getAddressBook(ctx.config.namespace)
-                ctx.config.valuesArg += ` --set "hedera-mirror-node.importer.addressBook=${ctx.addressBook}"`
-              }
-            },
-            {
-              title: `Upgrade chart '${constants.FULLSTACK_DEPLOYMENT_CHART}'`,
-              task: async (ctx, _) => {
-                await this.chartManager.upgrade(
-                  ctx.config.namespace,
-                  constants.FULLSTACK_DEPLOYMENT_CHART,
-                  ctx.config.chartPath,
-                  ctx.config.valuesArg
-                )
-              }
-            },
-            {
-              title: 'Waiting for explorer pod to be ready',
-              task: async (ctx, _) => {
-                if (ctx.config.deployHederaExplorer) {
-                  await this.k8.waitForPod(constants.POD_STATUS_RUNNING, [
-                    'app.kubernetes.io/component=hedera-explorer', 'app.kubernetes.io/name=hedera-explorer'
-                  ], 1, 100)
+          if (ctx.config.deployMirrorNode) {
+            const subTasks = [
+              {
+                title: 'Wait for proxies to verify node servers are running',
+                task: async (ctx, _) => {
+                  await sleep(15000) // give time for haproxy to detect that the node server is up on grpc port, queries every 10 seconds
+                }
+              },
+              {
+                title: 'Get the mirror node importer address book',
```

Comment:
`Get the mirror node importer address book` -> `Prepare address book`

Reply:
done